weaviate/tests/test_e2e.py

```diff
@@ -0,0 +1,24 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import pytest
+
+from datadog_checks.base.constants import ServiceCheck
+from datadog_checks.dev.utils import get_metadata_metrics
+
+from .common import E2E_METRICS, FLAKY_E2E_METRICS
+
+
+@pytest.mark.e2e
+def test_e2e_openmetrics_v1(dd_agent_check):
```

Comment:
why v1?

Reply:
Because 🤦 . I use some boilerplating when it comes to generating some of these tests. I'll have to update it. Should be v2 though.